implementations/rust/ockam/ockam_command/src/docs.rs

```
@@ -35,45 +35,60 @@ static THEME: Lazy<Option<Theme>> = Lazy::new(|| {
     Some(theme)
 });
 
+fn is_markdown() -> bool {
+    match std::env::var("OCKAM_HELP_RENDER_MARKDOWN") {
```

We could refactor all the places where we read environment variables to act a bit more like a command line option library:

 - with parsers giving you back a `bool`, a `String`, even a `Vec<T>` when `T` has a parser and we read comma-delimited values for example. This way we are alway consistent with how we encode `true` in environment variables
 - where we return a `Result` if something cannot be parsed. For example if I enter `OCKAM_HELP_RENDER_MARKDOWN=trua` (typo) I will get an error right away and not some silent default behaviour
 - with default values to be more explicit about what `Err(_e) => false` means

---

That sounds awesome, It would also be good to have a place in help that shows the environment variables that can be set.

---

Great idea, I created an issue for it: https://github.com/build-trust/ockam/issues/4554